doc/examples/config/config-storage/README.md

```diff
@@ -111,6 +111,18 @@ config.storage.put('/foo/bar', 'v2')
 ...
 ```
 
+Only in Enterprise Edition a TTL option is available. To set a TTL in seconds
```

Comment:
The config storage available only in Tarantool EE. I see no reason for this sentence.
```suggestion
To set a TTL in seconds
```

Reply:
Updated.